packages/material/scss/multiselect/_layout.scss

```diff
@@ -1,2 +1,9 @@
 @import "~@progress/kendo-theme-default/scss/multiselect/layout";
 
+@include exports("multiselect/layout/material") {
+
+    .k-button {
```

Comment:
This style is not scoped and reverts the `text-transform` from `button/_layout.scss`. It should either be scoped to the multiselect, or both rules should be deleted.